### PR TITLE
Add advanced qemu configuration Part 1

### PR DESCRIPTION
**Pull request by Quackdoc**
+ a small audio fix

this is some more advanced qemu configuration that people might like to utilize to get better performance out of their VMs, and/or better integration. 

needs proof read and tested. im sure there are many small grammer mistakes, and better wording I could have used, however the technical stuff should be accurate.

### CODE REVIEW

**Comment by Quackdoc** [comment]
I plan on eventually breaking this into 3 articles One for shared information across OS, one for unix since osx bsd and linux all mostly share the same feature set, and one page for windows specifically since it has special considerations it needs.

File: Installation/install-in-a-virtual-machine/advanced-qemu-config.md [hunk 1]
```diff
@@ -0,0 +1,182 @@
+---
+layout: post
+title: 'Advanced tweaks for qemu'
+date: '2021-10-27'
+---
+# Advanced configuration for qemu
+
+Qemu is a very powerfull tool that may be confusing for some people. however when using it we can get some very detailed setups, and advanced setups, this page will document some more advanced features that folk may wish to utilize with qemu. the majority of this page will assume running on linux. many of these will not be compatible with windows.
```
Comment (guanzhangrtk) [inline Installation/install-in-a-virtual-machine/advanced-qemu-config.md]: QEMU is a very powerful tool that may be confusing for some users. When using it we can get some very detailed and advanced setups. This page will document some more advanced features that users may wish to utilize with QEMU. The majority of this page will assume the user is running on Linux as many of these options may not be compatible on Windows.

**Comment by electrikjesus** [comment]
Is this PR still a WIP?

**Comment by Quackdoc** [comment]
yes, there are a few updates pending for qemu that may be landing soon too. so im waiting to see what happens with that since it will effect some of the stuff in here

**Comment by electrikjesus** [comment]
Do you want me to merge what we have here and then just save those updates for version 2.0?

**Comment by Quackdoc** [comment]
there are a few missing things, ill try to clean it up a chunk tonight

**Comment by Quackdoc** [comment]
Squashed, and cleaned up a bit, not sure how to wire this up to make it work with the UI though

**Pull request merged**